lib/build.js

```diff
@@ -361,7 +361,11 @@ module.exports = {
             }.bind(this), 1000);
           } else {
             var t = this.targets[this.activeTarget];
-            this.errorMatcher.set(t.errorMatch, this.replace(t.cwd), this.buildView.output.text());
+            var errorMatch = t.errorMatch;
+            if (!(errorMatch instanceof Array)) {
```

Comment:
I think this could be handled in the `set` function of `ErrorMatcher`. Also, it's probably cleaner to write `var errorMatch = (errorMatch instanceof Array) ? errorMatch : [ errorMatch ]`.